docs/reference/connector/docs/release-notes/connectors-release-notes-8.16.0.asciidoc

```diff
@@ -0,0 +1,46 @@
+[[es-connectors-release-notes-8-16-0]]
+=== 8.16.0 connectors release notes
+
+[discrete]
+[[es-connectors-release-notes-8-16-0-breaking]]
+==== Breaking Changes
+
+* Directly accessing connector and sync job state through `.elastic-connectors*` indices is deprecated, and will be disallowed entirely in a future release. Instead, the Elasticsearch Connector APIs should be used. Connectors framework code now uses the Elasticsearch Connector APIs by default.
+See https://github.com/elastic/connectors/pull/2884[*PR 2902*]..
+
+[discrete]
+[[es-connectors-release-notes-8-16-0-enhancements]]
+==== Enhancements
+
+* *Docker namespaces*
++
+Starting in `8.16.0`, two of our published Docker images and release artifacts now _also_ live under a new namespace: `integrations`, in addition to the existing `enterprise-search` namespace.
+This concerns the https://github.com/elastic/connectors[Elastic Connectors] and the https://github.com/elastic/data-extraction-service[Elastic Data Extraction Service].
++
+Please note that we recommend using the new `integrations` namespace, as we are consolidating integration Docker images there. We encourage users to adopt the new namespace as soon as possible.
++
+** *Example*:
++
+Old namespace::
+`docker.elastic.co/enterprise-search/elastic-connectors:v8.16.0`
```

Comment:
We should similarly note that the enterprise-search namespace is deprecated, and will eventually not house new connector releases, while the integrations space will.